drivers/storage/libstorage/libstorage_driver_tls.go

```diff
@@ -45,49 +45,49 @@ func verifyKnownHost(
 				"actualHost":          cert.Subject.CommonName,
 				"expectedHost":        knownHost.Host,
 			}).Debug("matched tls known host information")
-			return nil
+			return true, nil
 		}
 	}
-	return errServerFingerprint
+	return false, newErrKnownHost(peerCerts)
 }
 
 func verifyKnownHostFiles(
```

Comment:
Hi @vladimirvivien,

Thank you for the suggestion. The patch doesn't include the addition of these functions as they're not new. They're also private, so I think it's not as important to name them completely different. Their purpose is pretty obvious since the first one is 22 lines long and calls the second twice, once for each known_hosts file. I'm going to leave them as is for now for these reasons.